checks/evaluation/maintained.go

```diff
@@ -24,9 +24,10 @@ import (
 )
 
 const (
-	lookBackDays    = 90
-	activityPerWeek = 1
-	daysInOneWeek   = 7
+	lookBackDays     = 90
+	activityPerWeek  = 1
+	daysInOneWeek    = 7
+	newProjectWeight = -3
```

Comment:
We should zero out the score if less than 90 days, not just -3. We can't trust if earlier than 90 days.

Reply:
>We should zero out the score if less than 90 days, not just -3. We can't trust if earlier than 90 days.

Why zero out the score? This means any new repository would get `0` score.

Reply:
> > We should zero out the score if less than 90 days, not just -3. We can't trust if earlier than 90 days.
> 
> Why zero out the score? This means any new repository would get `0` score.

Yes that should be expected. We can't trust any new repository from day 1, its content needs to be evaluated and it needs to take time to build trust. Also, this will be just one check failure, rest everything should pass, so upto consumer to decide.

Reply:
Updated wording to make this a bit more clear - being younger than 90 days makes it harder for us to assess the history of maintenance of a project